recipes/paho-mqtt-c/all/conanfile.py

```diff
@@ -33,9 +32,9 @@ def _source_subfolder(self):
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
-        # Static linking before 1.3.4 isn't supported
-        if tools.Version(self.version) < "1.3.4":
-            self.options.shared = True
+            # Static linking before 1.3.4 isn't supported on Windows
```

Comment:
I think we had discussions about this before and it is unrelated to fixing the current problem. I'm not going to approve this.

Reply:
right , this discussion with the awesome library author who does not know what he actually delivers.
and where you said what I actually had running did not work :-) 

No problem, I will change it back so you feel better,  I have my own branch anyway (luckily, otherwise I would not have been able to create that fix for your changes so fast) so I do not care if this not so good change break history of things that previously worked, namely static libs for older versions on Linux and OSX 

The absurdity around this recipe can not shock or annoy me anymore, so I will give you the favour, even if I would have preferred you simply submit an edit to the PR to spare me the work